physics/bl_mynn_common.f90

```diff
@@ -22,22 +22,22 @@ module bl_mynn_common
  save
 
 ! To be specified from dycore
- real(kind=kind_phys):: cp           != 7.*r_d/2. (J/kg/K)
- real(kind=kind_phys):: cpv          != 4.*r_v    (J/kg/K) Spec heat H2O gas
- real(kind=kind_phys):: cice         != 2106.     (J/kg/K) Spec heat H2O ice
- real(kind=kind_phys):: cliq         != 4190.     (J/kg/K) Spec heat H2O liq
- real(kind=kind_phys):: p608         != R_v/R_d-1.
- real(kind=kind_phys):: ep_2         != R_d/R_v
- real(kind=kind_phys):: grav         != accel due to gravity
- real(kind=kind_phys):: karman       != von Karman constant
- real(kind=kind_phys):: t0c          != temperature of water at freezing, 273.15 K
- real(kind=kind_phys):: rcp          != r_d/cp
- real(kind=kind_phys):: r_d          != 287.  (J/kg/K) gas const dry air
- real(kind=kind_phys):: r_v          != 461.6 (J/kg/K) gas const water
- real(kind=kind_phys):: xlf          != 0.35E6 (J/kg) fusion at 0 C
- real(kind=kind_phys):: xlv          != 2.50E6 (J/kg) vaporization at 0 C
- real(kind=kind_phys):: xls          != 2.85E6 (J/kg) sublimation
- real(kind=kind_phys):: rvovrd       != r_v/r_d != 1.608
+ real(kind=kind_phys):: cp           !<= 7.*r_d/2. (J/kg/K)
```

Comment:
@mzhangw I understand that you have allowed these variables to be parsed by Doxygen now, but in the HTML output, it doesn't look like this is connected to anything. That is, if one goes to the HTML for the MYNN scheme, I can't see where to navigate to to find these. If you explicitly search for `bl_mynn_common`, it'll bring up a page with these variables, but as far as I can tell, that's the only way to see them. By the way, I'm not saying it's absolutely necessary for users to see this, I'm just pointing it out in case you intended to have them visible from the MYNN page or something.

Reply:
Thanks. It is also shown on the MYNN-EDMF module page now.